project/Dependencies.scala

```diff
@@ -7,23 +7,36 @@ import sbt._
   */
 object Dependencies {
 
-  val scalatest = "org.scalatest" %% "scalatest" % "3.0.4" % "test"
-  val avro = "org.apache.avro" % "avro" % "1.8.2"
-  val typesafe_config = "com.typesafe" % "config" % "1.3.1"
-  val avro4s = "com.sksamuel.avro4s" %% "avro4s-core" % "1.8.3"
-  val hbase_server = "org.apache.hbase" % "hbase-server" % "1.2.0" % "provided"
-  val hbase_common = "org.apache.hbase" % "hbase-common" % "1.2.0" % "provided"
-  val hadoop_common = "org.apache.hadoop" % "hadoop-common" % "2.6.0" % "provided"
-  val reflections = "org.reflections" % "reflections" % "0.9.11"
-  val spark_core = "org.apache.spark" %% "spark-core" % "2.3.0" % "provided"
-  val spark_sql = "org.apache.spark" %% "spark-sql" % "2.3.0" % "provided"
-  val postgres_conn = "org.postgresql" % "postgresql" % "9.3-1100-jdbc4"
-  val junit = "org.junit.jupiter" % "junit-jupiter-api" % "5.3.2" % Test
+  lazy val scalatest = "org.scalatest" %% "scalatest" % "3.0.4" % "test"
```

Comment:
changed val to lazy val to let the compiler figure out the initialization order of variables